### PR TITLE
NEW: Adds Authentication guide using V1 endpoint

### PR DESCRIPTION
**Pull request by glennr**

### CODE REVIEW

**Comment by glennr** [comment]
[Authentication / Log in (1)](https://trello.com/c/FGFNcuEr/40-authentication-log-in-1)

**Comment by glennr** [comment]
[Getting Started: Authentication (0.5)](https://trello.com/c/2dO7dQPW/47-getting-started-authentication-05)

File: api/authentication.md [hunk 1]
```diff
@@ -5,3 +5,87 @@ nav_order: 1
...
+
+Which yields a response payload with your user profile and token like so:
+
+```
+{
+    "email": "glenn@groovehq.com",
+    "first_name": "Glenn",
+    "last_name": "Groove",
+    "role": "admin",
+    "token": "2790383858d0a333dd...",
```
Comment (marcinbunsch) [inline api/authentication.md:50]: How about 
```suggestion
    "token": "<ACCESS_TOKEN>,
```

That way it's consistent with the later example

**Review by marcinbunsch — APPROVED** [review]
Small suggestion, lgtm

**Pull request merged**